eng/Versions.props

```diff
@@ -26,6 +26,10 @@
     <MicrosoftNETWorkloadEmscriptennet6Manifest70100Version>7.0.1</MicrosoftNETWorkloadEmscriptennet6Manifest70100Version>
     <MicrosoftNETWorkloadEmscriptennet7Manifest70100Version>7.0.1</MicrosoftNETWorkloadEmscriptennet7Manifest70100Version>
   </PropertyGroup>
+  <ItemGroup>
+    <!-- The bands we want to produce workload manifests for -->
```

Comment:
```suggestion
    <!-- The bands we want to produce workload manifests for.
         `SupportsMachineArch`: Metadata describing whether the VS authoring supports the machineArch SWIX property.
    -->
```

I found that in

https://github.com/dotnet/arcade/blob/14468017b1f17d4f780aec3b0b3d20511fef308b/src/Microsoft.DotNet.Build.Tasks.Workloads/src/Metadata.cs#L41-L44

Reply:
that would be good for main